README.md

```diff
@@ -1,79 +1,44 @@
 # Quasar S3 Datasource
 
-[![Build Status](https://travis-ci.org/slamdata/quasar-s3.svg?branch=master)](https://travis-ci.org/slamdata/quasar-s3)
+[![Build Status](https://travis-ci.org/slamdata/quasar-datasource-s3.svg?branch=master)](https://travis-ci.org/slamdata/quasar-datasource-s3)
 
-A datasource for the Quasar open source analytics engine, that
-provides access to Amazon S3.
+## Usage
 
-## How to use this
-
-1. Clone this repository (`git@github.com:slamdata/quasar-s3.git`)
-2. At the root of the repo, run `./sbt assembleDatasource`. This will generate a tarball with a loadable `slamdata-backend` plugin
-in `.targets/datasource/scala-2.12/quasar-s3-<version>-explode.tar.gz`
-3. Extract the tarball to SlamData Backend's plugin directory. By default that is `$HOME/.config/slamdata/plugin/`
-4. Run SlamData backend and the datasource should be available
+```sbt
+libraryDependencies += "com.slamdata" %% "quasar-datasource-s3" % <version>
+```
 
 ## Configuration
 
-You can create a new S3 datasource after you've loaded this plugin into
```

Comment:
👍

I originally added these when the API didn't have documentation. No longer necessary.